askama_derive/src/input.rs

```diff
@@ -1055,7 +1055,7 @@ const _: () = {
 };
 
 #[cfg(feature = "code-in-doc")]
-const JINJA_EXTENSIONS: &[&str] = &["askama", "j2", "jinja", "jinja2", "rinja"];
+const JINJA_EXTENSIONS: &[&str] = &["askama", "j2", "jinja", "jinja2", "askama"];
```

Comment:
```suggestion
const JINJA_EXTENSIONS: &[&str] = &["askama", "j2", "jinja", "jinja2"];
```

Reply:
I'll actually keep rinja for memories. :)